schemas/EiffelMetaProperty/1.0.0.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$"
+    },
+    "type": {
+      "type": "string",
+      "pattern": "^Eiffel[a-zA-Z]*Event$"
```

Comment:
As a comment: As we no longer validate for actual event names, we should step the schema version. This is creating a weaker validation possibility. Can we send in parameters for this?

Reply:
I haven't found a way to express this with json-schema.

Reply:
@magnusbaeck I think you wanted to have a regexp here instead of listing all events. Isn't it safer to list the exact names as we now don't validate the names at all `EiffelActyvilyStratlyEvent` would be valid?